eslzArm/eslz-portal.json

```
@@ -2032,6 +2032,298 @@
+                            "label": "Hub Routing Preference",
+                            "defaultValue": "ExpressRoute",
+                            "multiselect": false,
+                            "selectAll": false,
+                            "filter": false,
+                            "multiLine": true,
+                            "visible":"[and(not(equals(steps('connectivity').enableHub, 'No')), not(equals(steps('connectivity').enableHub, 'nva')), not(equals(steps('connectivity').enableHub, 'vhub')))]",
+                            "toolTip":"Preference used in selecting best path when the virtual hub learns multiple paths to the same destination route-prefix.<a href=\"https://learn.microsoft.com/en-us/azure/virtual-wan/about-virtual-hub-routing-preference\">Virtual hub routing preference</a>.",
+                            "constraints":{ 
+                                "allowedValues":[
+                                    {
+                                        "label":"ExpressRoute",
+                                        "description":"ExpressRoute is the preferred path",
+                                        "value":"ExpressRoute"
+                                    },
+                                    {
+                                        "label":"VPN",
+                                        "description":"VPN is the preferred path",
+                                        "value":"VpnGateway"
+                                    },
+                                    {
+                                        "label":"AS Path",
+                                        "description":"AS Path is the preferred path",
+                                        "value":"ASPath"
+                                    }
+                                ]
+                            }
+                        },
+                        {
+                            "name": "vWANHubCapacity",
+                            "type": "Microsoft.Common.DropDown",
+                            "label": "Virtual Hub Capacity",
+                            "defaultValue": "",
+                            "multiselect": false,
+                            "selectAll": false,
+                            "filter": false,
+                            "multiLine": true,
+                            "visible":"[and(not(equals(steps('connectivity').enableHub, 'No')), not(equals(steps('connectivity').enableHub, 'nva')), not(equals(steps('connectivity').enableHub, 'vhub')))]",
+                            "toolTip": "Routing infrastructure units determine the minimum throughput of the Virtual WAN hub router and the number of Virtual Machines that can be deployed in Virtual Networks connected to the Virtual WAN hub. Two routing infrastructure units are included at no extra cost with a deployment of a hub.<a href=\"https://learn.microsoft.com/en-us/azure/virtual-wan/hub-settings#capacity\">Virtual Hub Capacity</a>.",
```

```suggestion
                            "toolTip": "Routing infrastructure units determine the minimum throughput of the Virtual WAN hub router and the number of Virtual Machines that can be deployed in Virtual Networks connected to the Virtual WAN hub. Two routing infrastructure units are included at no extra cost with a deployment of a hub.<a href=\"https://learn.microsoft.com/azure/virtual-wan/hub-settings#capacity\">Virtual Hub Capacity</a>.",
```
Removing the fixed language to make this more D&I

---

@jtracey93, thanks for the comments. Just fixed them as you've suggested and pushed the changes into my branch. Could you please review again?